src/components/CommentDeleter.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import { deleteComment } from "../api";
+import ArticleComments from "./ArticleComments";
+
+export default function CommentDeleter({
+  comment,
+  articleComments,
+  setArticleComments,
+  isCommentDeleted,
+  setIsCommentDeleted,
+}) {
+  const [commentAuthor, setCommentAuthor] = useState("jessjelly");
```

Comment:
This doesn't need to be state if it's just hardcoded it can be in a const, if you are up to the challenge you could add it to a `context` and then it would be available in all your app and you could add functionality to it later down the line